tests/installation/resolve_dependency_issues.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Unified dependency issues resolver
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use base "y2logsstep";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    assert_screen('installation-settings-overview-loaded', 90);
+    if (check_screen('dependency-issue', 0) && get_var("WORKAROUND_DEPS")) {
+        $self->workaround_dependency_issues;
+    }
+    if (check_screen('dependency-issue', 0) && get_var("BREAK_DEPS")) {
+        $self->break_dependency;
+    }
+    if (check_screen('manual-intervention', 0)) {
+        $self->deal_with_dependency_issues;
```

Comment:
just find out that `deal_with_dependency_issues` already have logic which operate with variables which you operate above , so this module creates pretty weird flow ...
if I am not missing something based on code what I see in `deal_with_dependecy_issues` there are two possibilities : 
1. `WORKAROUND_DEPS` or `BREAK_DEPS` defined - they will be called second time . Not nice - something wrong with functions if we need to call them second time
2. Variables not define . you will die here . Not nice to hide this inside function . more clear just die directly here

Reply:
@asmorodskyi i have addressed this somehow. take a look when you have the time. looks to work but i think @rwx788 had some concerns about this approach.